docs/css/extra.css

```diff
@@ -2,6 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+h1 {
+  text-align: center;
+  margin-bottom: 1em;
+}
```

Comment:
This shouldn't be for all `h1` tags, since that'll look weird on actual documentation pages. (Personally, I don't think we need to center the heading for the download page either, but I don't have a super strong opinion.)

Reply:
The purpose of this change was not to affect everything necessarily, so I'll back it out for now.